synth.py

```diff
@@ -22,7 +22,7 @@
 
 gapic = gcp.GAPICBazel()
 common = gcp.CommonTemplates()
-versions = ["v1beta1", "v1beta2", "v1"]
+versions = ["v1"]
```

Comment:
Are we also removing the gapics for `v1beta1` and `v1beta2` or leaving those as-is? If they are staying, do we test these?

Reply:
It was just confirmed we're removing those - I'll follow up with deleting the corresponding code.